staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -128,6 +128,19 @@ const (
 	PodFailurePolicyOnExitCodesOpNotIn PodFailurePolicyOnExitCodesOperator = "NotIn"
 )
 
+// This field controls when we recreate pods
+// Default will be TerminatingOrFailed ie recreate pods when they are failed
+// +enum
+type PodReplacementPolicy string
+
+const (
+	// This is a field that recreates pods when they are marked as terminating or failed
```

Comment:
This is not a field, but a value.

You should start like "TerminatingOrFailed is a policy where..."